lib/components/hoc/WithScripts/WithScripts.story.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { defaultConfig, defaultConfig1 } from './WithScripts.mock';
+
+// Import Styled Component to showcase variations
+import withScripts from './WithScripts';
+
+const comp1 = () => (
+  <div>Hoc will consume the wrapper Component A and injected respective script details</div>
+);
+const comp2 = () => (
+  <div>Hoc will consume the wrapper Component B and injected respective script details</div>
+);
+
+const ComponentA = withScripts(comp1, defaultConfig);
+
+const ComponentB = withScripts(comp2, defaultConfig1);
+
+// $FlowFixMe eslint-disable-next-line extra-rules/no-commented-out-code
+storiesOf('remove duplicate script injection', module)
```

Comment:
@ritugupta64 We have removed the addWithChapters API please move the code to new format we are using. Merge the latest Develop and pick it from any component.

Reply:
Done